### PR TITLE
MP-5479 eclipse 2023-06

### PR DESCRIPTION
**Pull request by StefanVonDenBerken**

### CODE REVIEW

File: de.yatta.ideintruders.client/.classpath [hunk 1]
```diff
@@ -1,11 +1,11 @@
...
+<?xml version="1.0" encoding="UTF-8"?>
+<classpath>
+	<classpathentry kind="con" path="org.eclipse.jdt.launching.JRE_CONTAINER/org.eclipse.jdt.internal.debug.ui.launcher.StandardVMType/JavaSE-17">
+		<attributes>
+			<attribute name="module" value="true"/>
+		</attributes>
+	</classpathentry>
+	<classpathentry kind="con" path="org.eclipse.pde.core.requiredPlugins"/>
+	<classpathentry kind="src" path="src/"/>
+	<classpathentry kind="output" path="bin"/>
```
Comment (l3-g5) [inline de.yatta.ideintruders.client/.classpath:10]: why no longer target/classes?

File: de.yatta.ideintruders.client/IDE-Intruders.launch [hunk 2]
```diff
@@ -13,7 +13,22 @@
     <booleanAttribute key="clearwslog" value="false"/>
     <stringAttribute key="configLocation" value="${workspace_loc}/.metadata/.plugins/org.eclipse.pde.core/IDE-Intruders"/>
     <booleanAttribute key="default" value="false"/>
-    <setAttribute key="deselected_workspace_bundles"/>
+    <setAttribute key="deselected_workspace_bundles">
```
Comment (l3-g5) [inline de.yatta.ideintruders.client/IDE-Intruders.launch:16]: unneeded for this project

File: de.yatta.ideintruders.client/IDE-Intruders.launch [hunk 3]
```diff
@@ -238,23 +210,23 @@
...
         <setEntry value="org.tukaani.xz@default:default"/>
         <setEntry value="org.w3c.css.sac@default:default"/>
         <setEntry value="org.w3c.dom.events@default:default"/>
         <setEntry value="org.w3c.dom.smil@default:default"/>
         <setEntry value="org.w3c.dom.svg@default:default"/>
-        <setEntry value="slf4j.api@default:default"/>
-        <setEntry value="slf4j.nop@default:default"/>
     </setAttribute>
     <setAttribute key="selected_workspace_bundles">
+        <setEntry value="com.yattasolutions.platform.marketplace.client@default:default"/>
```
Comment (l3-g5) [inline de.yatta.ideintruders.client/IDE-Intruders.launch:223]: please update this file with standalone eclipse

File: de.yatta.ideintruders.client/.classpath (hunk 1 above)
Comment (StefanVonDenBerken) [reply de.yatta.ideintruders.client/.classpath:10]: That is changed by eclipse on every "maven update", when imported via "existing projects into workspace". It is not changed, when importing as maven project, but then the plugin development is not working.

**Pull request merged**